.github/workflows/lint-and-test.yml

```diff
@@ -1,25 +1,30 @@
-name: Lint and test
+name: Test
 
 on: [push, pull_request]
 
 jobs:
-  build_and_test:
+  docker:
+    timeout-minutes: 10
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        nodejs: [10, 12, 14]
 
     steps:
-    - uses: actions/checkout@v2
+    - name: Checkout
+      uses: actions/checkout@v1
 
-    # https://github.com/actions/setup-node
-    - uses: actions/setup-node@v2-beta
+    - name: Start containers
+      run: docker-compose -f "docker-compose.yml" up -d mysql
+
+    - name: Install node
+      uses: actions/setup-node@v1
       with:
-        node-version: ${{ matrix.nodejs }}
+        node-version: 14.x
+
+    - name: Install dependencies
+      run: npm install
+
+    - name: Run tests
+      run: DOTENV=.env.local npx jest
```

Comment:
Instead of doing this way, shall we specify the env variable as in here https://docs.github.com/en/actions/reference/environment-variables